OKTA-1082814 - Adding API release notes 2026.05.0 (#6139)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2026-okta-identity-engine/index.md

| [Service accounts is GA in Preview](#service-accounts-is-ga-in-preview) | May 6, 2026 |
| [PIV Identity Provider JIT provisioning updates](#piv-identity-provider-jit-provisioning-updates) | May 6, 2026 |
| [PIV Identity Provider supports dynamic user matching](#piv-identity-provider-supports-dynamic-user-matching) | May 6, 2026 |
| [SHA-256 digest algorithm support is GA in Preview](#sha-256-digest-algorithm-support-is-ga-in-preview) | Dec 10, 2025 |
| [Skip counts for authenticator enrollment grace periods is GA in Production](#skip-counts-for-authenticator-enrollment-grace-periods-is-ga-in-production) | Feb 4, 2026 |
| [Client update policy is GA in Production](#client-update-policy-is-ga-in-production) | Jan 7, 2026 |
| [Passkeys rebrand is GA in Production](#passkeys-rebrand-is-ga-in-production) | Feb 4, 2026 |
| [Developer documentation updates in 2026.05.0](#developer-documentation-updates-in-2026-05-0) | May 6, 2026 |
| [Bug fixed in 2026.05.0](#bug-fixed-in-2026-05-0)| May 6, 2026 |

#### Identity claims sourcing policy is self-service EA in Preview

> You can now configure the identity claims sourcing policy which enables federated users to re-authenticate with their most recent and active SSO IdP.

Reviewer: ```suggestion
You can now configure the identity claims sourcing policy which enables federated users to reauthenticate with their most recent and active SSO IdP.
```

> Reauthentication to an IdP helps Okta admins secure federated identities by redirecting federated users to their source SAML, OIDC, or Org2Org IdP when a policy requires them to reauthenticate. By forcing reauthentication at the source IdP, admins can close security gaps from long-lived sessions and remove the need to configure duplicate MFA enrolment in Okta.

Reviewer: ```suggestion
Reauthentication to an IdP helps Okta admins secure federated identities by redirecting federated users to their source SAML, OIDC, or Org2Org IdP when a policy requires them to reauthenticate. By forcing reauthentication at the source IdP, admins can close security gaps from long-lived sessions and remove the need to configure duplicate MFA enrollment in Okta.
```

<!-- OKTA-1169086 EMAIL_ENROLLMENT_AND_RECOVERY_CONTROL preview date: may 6, 2026 -->

#### Managed connections renamed to resource connections is GA in Production

The **Managed connections** tab has been renamed to **Resource connections** on the Okta for AI Agents pages. The associated System Logs' event types, debug data key, and details entries have been renamed accordingly.

#### Service accounts is GA in Preview

The Okta Managed User Accounts API is now available for Okta Privileged Access-enabled orgs. Okta Privileged Access secures SaaS service accounts that allows customers to monitor, manage, and control access to service accounts in their SaaS apps. Okta users designated with privileged access are treated as service accounts that resource admins can assign to resource groups and projects, and security admins can create policies to configure which users can access them. The Okta Managed User Accounts API provides operations to manage these user accounts in Okta Universal Directory with OPA.

> This feature is available only if you're subscribed to Okta Privileged Access. Ensure that you've set up the Okta Privileged Access app before creating app accounts through the API.

Reviewer: add a blank line between this paragraph and the first one

>  See [Okta Managed User Accounts](https://developer.okta.com/docs/api/openapi/okta-management/management/tags/oktamanageduseraccount), [Service Accounts](https://developer.okta.com/docs/api/openapi/okta-management/management/tags/serviceaccount) and [Manage service accounts](https://help.okta.com/okta_help.htm?type=oie&id=ud-privileged-acnt).

Reviewer 2: This API is not yet live. Don't we need to comment it out? 

`[Okta Managed User Accounts](https://developer.okta.com/docs/api/openapi/okta-management/management/tags/oktamanageduseraccount). `

Same comment for Classic and PAM RN.

Author: Yes, I'll remove the links. And I presume I keep this RN in all three locations.

This feature allows admins to define a number of skips end users can defer enrollment into an authenticator, as well as customizations to the prompt when end users see the grace period. See [Grace periods](/docs/concepts/policies/#grace-periods) and [type](https://developer.okta.com/docs/api/openapi/okta-management/management/tags/policy/other/createpolicy#other/createpolicy/t=response&c=200&path=&d=1/settings/authenticators/enroll/graceperiod).

<!-- OKTA-1044803 FF: ENROLLMENT_POLICY_GRACE_PERIOD_V2 preview date: Feb 4, 2026 -->

#### Client update policy is GA in Production

The Policies API now supports the `CLIENT_POLICY` type, enabling you to enforce or defer app updates across different device platforms. This lets you programmatically align app versions with internal change management processes. See the [Policies API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/#tag/Policy/operation/listPolicies) and [Release controls policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-ov-release-controls). <!-- OKTA_VERIFY_RELEASE_CONTROL_POLICY OKTA-1036332 January 7, 2026 -->

#### Passkeys rebrand is GA in Production

> The FIDO2 (WebAuthn) authenticator is being rebranded to Passkey (FIDO2 WebAuthn) and Okta is introducing enhanced administrative controls and a streamlined user experience. This update centralizes passkey management through a consolidated settings page, allows for customized authenticator naming, and introduces a dedicated **Sign in with a passkey** button within the Sign-In Widget. These enhancements simplify the authentication journey and provide users with a more intuitive sign-in process with the **Sign in with a passkey** button.

Reviewer: ```suggestion
The FIDO2 (WebAuthn) authenticator is being rebranded to Passkey (FIDO2 WebAuthn), and Okta is introducing enhanced administrative controls and a streamlined user experience. This update centralizes passkey management through a consolidated settings page, allows for customized authenticator naming, and introduces a dedicated **Sign in with a passkey** button within the Sign-In Widget. These enhancements simplify the authentication journey and provide users with a more intuitive sign-in process with the **Sign in with a passkey** button.
```

For more information about the new settings and updates, see [Passkeys and WebAuthn](/docs/guides/authenticators-web-authn/aspnet/main/#passkeys-and-webauthn) and [Configure the Passkey (FIDO2 WebAuthn) authenticator](https://help.okta.com/okta_help.htm?type=oie&id=csh-configure-webauthn).

<!-- OKTA-1012303 FF: PASSKEYS_REBRAND preview date: Feb 4, 2026 -->

#### Developer documentation updates in 2026.05.0

* The new [Sign-up flows](/docs/concepts/sign-up-flows/ ) guide provides best practices for designing secure, high-conversion registration strategies. This resource helps developers balance user experience and security by covering topics such as progressive enrollment and automated group assignments.

> * The new [Script your user migration with the Okta Users API](/docs/guides/migrate-to-okta-with-scripts/) guide demonstrates how to programmatically import users from an external system to Okta using the Okta Users API. This guide provides best practices and sample code to help you script 3 import scenarios.

Reviewer: ```suggestion
* The new [Script your user migration with the Okta Users API](/docs/guides/migrate-to-okta-with-scripts/) guide demonstrates how to programmatically import users from an external system to Okta using the Okta Users API. This guide provides best practices and sample code to help you script three import scenarios.
```
